lib/ttnt/version.rb

```diff
@@ -1,3 +1,4 @@
 module TTNT
+  # TTNT version
```

Comment:
True. Sorry I added this in order to make documentation coverage (which you can produce with `yard stats`) 100%, but you are right it's useless :sweat_smile:  I removed it here: https://github.com/Genki-S/ttnt/commit/4ff64326dd2fdb830854f8d10dbded36520002c4